template/menu.phtml

```diff
@@ -11,7 +11,8 @@
         <ul class="menu-nav">
             <?php foreach ($menuitems as $url => $label): ?>
                 <li class="nav-item">
-                    <a href="<?= $url ?>" class="nav-link" <?= getCurrentPageId($url) ?>"><?= $label ?></a>
+                    <a href="index.php?page=<?= basename($url, '.php') ?>" class="nav-link <?= getCurrentPageId($url) ?>"><?= $label ?></a>

```

Comment:
Due to security its better to have parameters as text, not as the filename. So the URL will be index.php?page={pageName} where {pageName} won't have .php extension